client/layout/masterbar/masterbar-notifications/notifications-button.jsx

```diff
@@ -112,22 +115,14 @@ class MasterbarItemNotifications extends Component {
 			<>
 				<MasterbarItem
 					url="/notifications"
-					icon="bell"
+					icon={ <BellIcon newItems={ this.state.newNote } active={ this.props.isActive } /> }
 					onClick={ this.toggleNotesFrame }
 					isActive={ this.props.isActive }
 					tooltip={ this.props.tooltip }
 					className={ classes }
 					ref={ this.notificationLink }
-				>
-					{ this.props.children }
-					<span
-						className="masterbar__notifications-bubble"
-						key={
-							'notification-indicator-animation-state-' + Math.abs( this.state.animationState )
-						}
```

Comment:
This key prop should be set to the `BellIcon`. Changing the key causes re-rendering of the element, thus running the animation again. This is a nice way to show the bubble dancing when you get a new notification.